leapp/config.py

```diff
@@ -40,6 +40,9 @@
         'dir': '/var/log/leapp/',
         'files': ','.join(_FILES_TO_ARCHIVE),
     },
+    'lock': {
+        'path': '/var/lib/leapp/leapp_lock.pid'
+    },
```

Comment:
This will cause the lock file to moved on the first execution right away...

https://github.com/oamg/leapp-repository/blob/214ed9b57c5e291cda5ff6baf7c7a790038fef34/commands/upgrade/__init__.py#L81

Reply:
Combining this with the idea that the lockfile should not survive reboot, perhaps adding it to `/run` and letting it be cleared on reboot would be appropriate?  (I believe that /run is usually a tmpfs nowadays so that would protect it.  I don't recall if systemd also has a way to clear out stale lockfiles on boot if /run happens to not be a tmpfs.)

Reply:
/run is actually the place it should be at

Reply:
Yeah I definitely agree. This solves a lot of issues. Thanks!!